hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/debezium/DebeziumSource.java

```diff
@@ -239,5 +239,13 @@ private static Dataset<Row> convertArrayColumnsToString(Dataset<Row> dataset) {
 
     return dataset;
   }
+
+  @Override
+  public void onCommit(String lastCkptStr) {
```

Comment:
Looking at other sources realized that Debezium doesn't commit offsets currently. After confirming w/ @rmahindra123, decided to address this in that PR as it's a simple ~copy-paste~ change